build/yamls/base/conf/antrea-agent.conf

```diff
@@ -12,6 +12,10 @@ featureGates:
 # to define security policies which apply to the entire cluster.
 #  ClusterNetworkPolicy: false
 
+# Enable AntreaNetworkPolicy feature to complement K8s NetworkPolicy for security policies that
+# supports priorities, actions and externalEntities.
+#  AntreaNetworkPolicy: false
```

Comment:
Could we use one feature gate with ClusterNetworkPolicy? Or you think namespaced policy can introduce extra risks?

Reply:
we should use one feature gate .. otherwise too much complexity

Reply:
@Dyanngg and i discussed this feature gate name.. think we should rename this to something like AntreaPolicy or Policy to generalize the feature gate name for CNP, ANP, and Tier etc.. what do you think @jianjuns @tnqn ?

Reply:
AntreaPolicy sounds good to me.